### PR TITLE
Integration Testing doc update

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Updated docs and samples for RTM. 

review @Tratcher @muratg 
docs @Rick-Anderson @rachelappel @danroth27

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
looks good to me.

**Comment by JunTaoLuo** [comment]
should I take that as a :shipit: ? The branch is squashed. Feel free to merge it if you think it's ready.

File: aspnet/testing/integration-testing/sample/global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "projects": [ "src", "test" ],
   "sdk": {
-    "version": "1.0.0-rc1-update1"
+    "version": "1.0.0-preview2-003121"
```
Comment (Tratcher) [inline aspnet/testing/integration-testing/sample/global.json]: Is this the final preview2 version?

Comment (danroth27) [inline aspnet/testing/integration-testing/sample/global.json]: Yup

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**